src/js/components/molecules/list-section/list-section.ts

```diff
@@ -87,7 +87,11 @@ export default class ListSection extends Vue implements IListSection {
 
     triggerEventClick (): void {
         if (this.expand && this.expand.clickEvent) {
-            this.$emitPublic(this.clickEventType, this.clickEventData);
+            try {
```

Comment:
ja, don't like the idea of swallowing the error. we're missing regression testing as well